package.json

```diff
@@ -40,7 +40,7 @@
     "node-sass": "^4.5.3",
     "normalize.css": "^7.0.0",
     "nunjucks": "^3.0.1",
-    "postcodesio-client": "^0.1.2",
+    "postcodesio-client": "neilbmclaughlin/postcodesio-client-node#feature\/add-outcode-lookup",
```

Comment:
Yep, if I did this again I'd have the fork in nhsuk. I'm hoping it is short lived.